class.photon.php

```diff
@@ -695,6 +700,12 @@ public function filter_srcset_array( $sources, $size_array, $image_src, $image_m
 			$newsources = null;
 
 			foreach ( $multipliers as $multiplier ) {
+
+				/** This filter is already documented in class.photon.php */
+				if ( apply_filters( 'jetpack_photon_skip_image', false, $url ) ) {
```

Comment:
That makes sense. What should I check for in the filter at that point? `trailingslashit( $upload_dir['baseurl'] ) . $image_meta['file']`?